bin/kibana

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-exec "${NODE}" $NODE_OPTIONS "${DIR}/src/cli" ${@}
```

Comment:
I think adding the flag all the time is fine, since we are forwarding all the messages via the server logger.